pkg/ha/keepalive.go

```diff
@@ -112,6 +126,30 @@ func KeepAlive(ctx context.Context, cli *clientv3.Client, workerName string, kee
 		case <-ctx.Done():
 			log.L().Info("ctx is canceled, keepalive will exit now")
 			return nil
+		case newTTL := <-keepAliveUpdateCh:
+			// create a new lease with new TTL, and overwrite original KV
+			cliCtx, cancel := context.WithTimeout(ctx, etcdutil.DefaultRequestTimeout)
+			defer cancel()
+			lease, err = cli.Grant(cliCtx, newTTL)
```

Comment:
The revokeLease function should be called for the old lease, like line 104.

Reply:
lease will expire if not keepalive, so I think it's not needed